server/chat/backend/agent/tools/github_rca_tool.py

```
@@ -691,6 +700,33 @@ def github_rca(
+    repo_full_name = f"{owner}/{repo_name}"
+    try:
+        auth = get_auth_for_user_repo(user_id, repo_full_name)
+    except NoGitHubAuthError:
+        return json.dumps({
+            "status": "error",
+            "error": "GitHub not connected for this user. Install the GitHub App or connect via OAuth.",
+            "action": action,
+            "repository": repo_full_name,
+        })
+    except Exception as e:
+        logger.error(
```

Ruff (0.15.12)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)